inputs/wysiwyg.vue

```diff
@@ -389,7 +271,8 @@
       }
     },
     mounted() {
-      const isSingleLine = this.isSingleLine,
+      const THIS = this,
+        isSingleLine = this.isSingleLine,
         isMultiLine = this.isMultiLine,
         isMultiComponent = this.isMultiComponent,
```

Comment:
What do you think if we destructure the `this` object to get all of its properties? For example:
```js
const {
    isSingleLine,
    isMultiLine,
    isMultiComponent
    // all the other props...
  } = this,
  // Anything else...
```
Also, why do we create a copy of `this` into `THIS`? Does it bring any benefits?

Reply:
👍 to the destructuring.  The THIS copy has already been eliminated.  Quill was losing the THIS where I needed it, but I found a way around it.